### PR TITLE
airplane icon

### PR DESCRIPTION
**Pull request by tyrellshawn**
## I have read the [CONTRIBUTING.md](https://github.com/pqoqubbw/icons/blob/main/CONTRIBUTING.md) file.

y

## What kind of change does this PR introduce?

feature

## Demo

Please attach a short video demo of the changes.

https://github.com/user-attachments/assets/4cd018c8-c2e2-4275-8540-ad8c573a3c4b

## Additional context

Add any other context or screenshots.

### CODE REVIEW

**Comment by tyrellshawn** [comment]
@pqoqubbw please review

File: icons/airplane.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+  const handleHover = () => {
+    setIsHovered(true);
+    controls.start('hover');
+  };
+
+  const handleHoverEnd = () => {
+    setIsHovered(false);
+    controls.start('initial');
+  };
```
Comment (pqoqubbw) [inline icons/airplane.tsx:18]: is there a possible solution to get rid of one of the states?

File: icons/airplane.tsx
Comment (pqoqubbw) [inline icons/airplane.tsx:1]: could you also move repeated value to variables? like for other icons

Comment (tyrellshawn) [reply icons/airplane.tsx:1]: I have applied both changes: 
- Now there is only one state
- I have captured reused code into variables

**Review by pqoqubbw — APPROVED** [review]

**Pull request merged**